stroeer/core/v1/core_article_service.proto

```diff
@@ -10,10 +10,42 @@ option java_package = "de.stroeer.core.v1";
 
 // Service to manage core articles
 service ArticleService {
-  // Returns a core article by it's id
+  // Returns a core article by it's `id`
   rpc GetArticle (GetArticleRequest) returns (stroeer.core.v1.Article) {}
+
+  // Returns pages of articles by their `home_section`
+  rpc ListArticles (ListArticlesRequest) returns (ListArticlesResponse) {}
+  // also as continuous stream
+  rpc LotsOfArticles (ListArticlesRequest) returns (stream ListArticlesResponse) {}
 }
 
 message GetArticleRequest {
   string id = 1;
 }
+
+message ListArticlesRequest {
+  // mandatory: valid section_path, with leading and trailing slash
+  // e.g. `/sport/`
+  string section_path = 1;
+
+  // The maximum number of books to return. The service may return fewer than
+  // this value.
+  // If unspecified, at most 10 articles will be returned.
+  // The maximum value is 50; values above 1000 will be coerced to 1000.
+  int32 page_size = 2;
+
+  // A page token, received from a previous `ListArticles` call.
+  // Provide this to retrieve the subsequent page.
+  //
+  // When paginating, all other parameters provided to `ListArticles` must match
+  // the call that provided the page token.
+  string page_token = 3;
+}
+
+message ListArticlesResponse {
+  repeated stroeer.core.v1.Article articles = 1;
+
+  // A token that can be sent as `page_token` to retrieve the next page.
+  // If this field is omitted, there are no subsequent pages.
```

Comment:
```suggestion
  // If this field is empty, there are no subsequent pages.
```

Reply:
this is official google slang copied from https://google.aip.dev/158